src/Symfony/Component/Validator/Mapping/Loader/XmlFileLoader.php

```diff
@@ -62,6 +50,20 @@ public function loadClassMetadata(ClassMetadata $metadata)
     }
 
     /**
+     * Return the names of the classes mapped in this file.
+     *
+     * @return string[] The classes names
+     */
+    public function getMappedClasses()
```

Comment:
Not worth it imho, we don't need to open extensibility here, we have Xml & Yaml loader, let's way until others of the same kind arrive (I doubt any will soon)